TUTORIAL.md

```diff
@@ -434,6 +436,20 @@ end
 
 For an argument, each function will be tried in order. The first function that doesn't raise a `ContractError` will be used. So in this case, if x == 1, the first function will be used. For all other values, the second function will be used.
 
+This allows you write methods more declaratively, rather than using conditionals. This feature is not only useful for recursion; you can use it to keep parallel use cases separate:
+
+```ruby
+Contract lambda{|n| n < 12 } => Ticket
+def get_ticket(age)
+  ChildTicket.new(age: age)
+end
+
+Contract lambda{|n| n >= 12 } => Ticket
```

Comment:
Do you mean combining contracts like `And[Num, lambda{|n| n < 12 }] => Ticket` ? I can change that. I think this is a good change, becaues it encourages people to write stricter contracts.